src/Tribe/Admin/Notice/Rest_Api.php

```diff
@@ -0,0 +1,122 @@
+<?php
+/**
+ * Shows an admin notice when our REST API endpoints are not available.
+ *
+ * @since
+ *
+ * @package Tribe\Events\Admin\Notice
+ */
+
+namespace Tribe\Events\Admin\Notice;
```

Comment:
Check if these can be created on the `TEC\Events\Admin\Notice` endpoint, it will be in a diff folder, but we should try to add any new classes over there.